app/views/projects/index.html.erb

```diff
@@ -14,9 +14,14 @@
         <div class="govuk-summary-list__row">
           <dt class="govuk-summary-list__key">
             <h2 class="govuk-heading-m govuk-heading-m--school-name">
-              <span><%= link_to "URN #{project.urn}", project_path(project) %></span>
+              <%= link_to "#{establishment_display_name(project.establishment)}", project_path(project) %>
+              <span class="govuk-caption-l inline"><%= "- #{project.urn}" %></span>
             </h2>
           </dt>
+          <dd class="govuk-summary-list__actions">
+            <strong class="govuk-tag govuk-tag--purple">Post Advisory Board
```

Comment:
Might be worth making this a locale string, I can see us using this in a few places